funnel/forms/profile.py

```diff
@@ -48,3 +53,43 @@ class ProfileTransitionForm(forms.Form):
 
     def set_queries(self):
         self.transition.choices = list(self.edit_obj.state.transitions().items())
+
+
+@Profile.forms('logo')
+class ProfileLogoForm(forms.Form):
+    logo_url = forms.URLField(
+        __("Logo URL"),
+        description=__("Profile logo"),
+        validators=[
+            forms.validators.Optional(),
+            forms.validators.ValidUrl(
+                allowed_schemes=lambda: current_app.config.get(
+                    'IMAGE_URL_SCHEMES', ('https',)
+                ),
+                allowed_domains=lambda: current_app.config.get('IMAGE_URL_DOMAINS'),
+                message_schemes=__("A https:// URL is required"),
+                message_domains=__("Images must be hosted at images.hasgeek.com"),
+            ),
+            forms.validators.Length(max=2000),
+        ],
+    )
+
+
+@Profile.forms('banner_image')
+class ProfileBannerForm(forms.Form):
+    banner_image_url = forms.URLField(
+        __("Banner image URL"),
+        description=__("URL for profile banner image"),
```

Comment:
This description asks for a "URL", but the logo field's description doesn't. Need consistency in language.